rhel9/Makefile

```diff
@@ -1,13 +1,16 @@
 PACKER ?= packer
+PACKER_LOG ?= 0
 ISO ?= rhel-baseos-9.1-x86_64-dvd.iso
 
 .PHONY: all clean
 
 all: rhel9.tar.gz
 
-rhel9.tar.gz: clean
-	sudo ${PACKER} init rhel9.pkr.hcl && sudo PACKER_LOG=1 ${PACKER} build -var "rhel9_iso_path=${ISO}" rhel9.pkr.hcl
-	reset
+check-deps:
+	dpkg -s libnbd-bin nbdkit packer fuse2fs > /dev/null
+
+rhel9.tar.gz: check-deps clean
+	sudo ${PACKER} init rhel9.pkr.hcl && sudo PACKER_LOG=${PACKER_LOG} ${PACKER} build -var "rhel9_iso_path=${ISO}" rhel9.pkr.hcl
```

Comment:
Why did you decrease this from 1 to 0? And maybe the variable could be names `PACKER_LOG_LEVEL` (I guess that's what it is, isn't it?)
(There is more of those below)

Reply:
`PACKER_LOG` is a boolean, and I changed it's value because verbose builds should not be the default. The output is not helpful unless you're making changes to the template, and it often corrupts the terminal.

Reply:
OK, +1 to this